test/distributed/test_c10d_gloo.py

```diff
@@ -236,8 +236,10 @@ def test_empty_tensors(self):
         pg = c10d.ProcessGroupGloo(store, self.rank, self.world_size, self.opts())
 
         xs = [torch.FloatTensor([])]
-        pg.broadcast(xs).wait()
-        self.assertEqual(0, xs[0].numel())
+        w = pg.broadcast(xs)
```

Comment:
Will the `output` be equal to `xs[0]` after `w.wait()`?

Reply:
added a check